reana_workflow_engine_yadage/cli.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # This file is part of REANA.
-# Copyright (C) 2017, 2018 CERN.
+# Copyright (C) 2017-2021 CERN.
```

Comment:
Cosmetics: In general, not every file was touched all the years. It'll be easier to stick to GNU copyright updating policy. Please see my comments here: https://github.com/reanahub/reana-client/pull/570#discussion_r724804703

Reply:
Will do it.